### DISCUSSION TITLE
HTML Terminado

### DISCUSSION THREAD

**Discussion by CRIS-117** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#279] [Show and tell]
Hola compañeros viajeros, espero se encuentren bien. Solo quería mostrarles lo que hice que por suerte acabe 2 horas antes, llevo 1 año aprendiendo y e tenido avances. 
Les comento como lo hice, busque imágenes de pasteles 3D en Google y me tope con esta pagina https://www.istockphoto.com/es/fotos/pasteles-3d las cuales descargue y después les quite el fondo con esta pagina https://www.remove.bg/es (gracias Tiktok) e hice mi logo en illistrator pero no lo agregue por que me gusto mas como lo hice con texto y un icono. 
Que les recomiendo:
Yo practique mucho con una youtuber donde el diseño de mi pagina es parecida a los suyos ya que hice varios de sus proyectos como copiloto, donde yo repito lo que ella hace mientras pasa el video. El canal de ella se llama [Bedimcode](https://www.youtube.com/c/Bedimcode) el cual les recomiendo.
También les recomiendo ver el canal de [Soy Dalto](https://www.youtube.com/c/soydalto) donde tiene curso de HTML, CSS y JS.

Bien sin mas que decir les dejo el link de mi pagina, un preview y mi logo que no salió a la luz.
https://cakecandle.000webhostapp.com/#
![preview](https://user-images.githubusercontent.com/90110151/156966432-cb4e7641-3634-4356-aaf9-1aa68e8ae81b.jpg)
![logo_1](https://user-images.githubusercontent.com/90110151/156966447-593a85bb-dfb3-4d19-835c-aa73459645c7.png)

**Comment by AngelAlfaro92** [comment]
Te quedó muy bien,
Disculpa, tienes algún tutorial que ya hayas usado tu para hacer el despliegue?? Justo terminé la práctica pero ya no me dio tiempo hacer el despliegue

**Reply by CRIS-117** [reply]
Hola, si lo puse en la descripción el canal con el que aprendí. :D

**Comment by hackit92** [comment]
Está excelente! Me parece un despliegue de muy buena calidad, lo único es que no encontré el acceso al dashboard del pastelero ni un formulario como tal para levantamiento de pedidos

**Reply by CRIS-117** [reply]
Si, como no ví los lives, no entendí esa parte hasta que ví sus trabajos, y pues ya no me dió tiempo, ya que también tengo tarea de mi ingeniería y también trabajo :( pero lo puedo agregar posteriormente. :3

**Reply by hackit92** [reply]
Si claro, perfectamente entendible y una vez mas... tu sitio luce excelente!

**Comment by JuanSalvador73** [comment]
Excelente, muchas felicidades por ese sitio, te quedó de 10. :D
